packages/turf-planepoint/README.md

```diff
@@ -4,24 +4,25 @@
 
 ## planepoint
 
-Takes a triangular plane as a [Polygon][1]
-and a [Point][2] within that triangle and returns the z-value
-at that point. The Polygon should have properties `a`, `b`, and `c`
+Takes a triangular plane as a polygon and a point within that triangle, and
+returns the z-value at that point.
+
+The Polygon should have properties `a`, `b`, and `c`
```

Comment:
I onboard with removing the links from the description for what it's worth.  It was inconsistently done and the parameters already provide it.